plugin/trino-druid/src/main/java/io/trino/plugin/druid/DruidJdbcClient.java

```diff
@@ -60,18 +60,18 @@
 import static io.trino.plugin.jdbc.StandardColumnMappings.timestampColumnMappingUsingSqlTimestampWithRounding;
```

Comment:
Make this commit the first commit - otherwise the actual commit fixing the issue does not compile cleanly (Checkstyle).

nit: Maybe rename to `Move public fields to the top of the class`

Reply:
I see, will exchange the commit orders. Or is it preferable to combine them into one commit?

Reply:
It's recommended to squash commits into one.

Reply:
Since this is a pure refactor I'd suggest to keep it separate. It's easier to see what's logical changes and what's just refactor that way.

Reply:
@hashhar agree